cinema/views.py

```diff
@@ -21,7 +23,7 @@
     MovieSessionDetailSerializer,
     MovieListSerializer,
     OrderSerializer,
-    OrderListSerializer,
+    OrderListSerializer, MovieImageSerializer,
```

Comment:
Code Style: Ensure that the last item in the import list is followed by a comma when using parentheses for grouping imports.